Install/batcenter_install.bat

```diff
@@ -24,8 +24,17 @@ REM Script to install BatCenter by Kvc
 Echo. FETCHING FILES FROM SERVER...
 
 IF /I "%_Curl%" == "YES" (
-    Curl -L --ssl-no-revoke "https://github.com/Batch-Man/BatCenter/blob/main/Files/7za.exe?raw=true" --output "7za.exe" --progress-bar
-    Curl -L --ssl-no-revoke "https://github.com/Batch-Man/BatCenter/blob/main/Install/bat.7z?raw=true" --output "bat.7z" --progress-bar
+    IF /I "%~1"=="debug" (

```

Comment:
what is this section for? I didn't get the purpose if it. @GroophyLifefor

Reply:
When I was downloading using curl, when I used your repo, what I added was not coming, so I added a parameter and I was debugging by downloading from my fork repo with this parameter, but there is no need for this in your repo.